### PR TITLE
remove unnecessary scroll from order form

### LINKED ISSUE #2568: In 'Guest Order Form' clicking on sign-in makes a unnecessary scroll
**Describe the bug**

Order a ticket as a guest, and enter the email of a pre-existing user and then click on sign-in.

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to 'Any Events Ticket'
2. Order Some tickets
3. Enter Email of a preexisting user, click on sign-in without filling the password field.

**Expected behaviour**

No scroll should be there
**Screenshots**

<a href="https://imgflip.com/gif/2ya62s"></a>

**Desktop (please complete the following information):**
 - OS:MacOS Mojave
 - Browser: Chrome
 - Version: 73.0.3

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Fixes #2568

### CODE REVIEW

**Comment by uds5501** [comment]
@kushthedude This does the job for us

File: app/components/forms/orders/guest-order-form.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import Component from '@ember/component';
 import FormMixin from 'open-event-frontend/mixins/form';
 
 export default Component.extend(FormMixin, {
+  autoScrollToErrors: false,
```
Comment (kushthedude) [inline app/components/forms/orders/guest-order-form.js]: Making just change to one place , i dont think this is the right solution @uds5501 , I was thinking to remove it from complete app

Comment (kushthedude) [reply app/components/forms/orders/guest-order-form.js]: It still solves #2568 But there are many other too 😅 ,Still for #2568 its the right solution , Lets see what mentors review , 👍

Comment (uds5501) [reply app/components/forms/orders/guest-order-form.js]: @kushthedude It is actually desirable to scroll up to that error. The problem in this particular case here is that this is a modal small enough to not require a scroll

**Comment by uds5501** [comment]
@CosmicCoder96 Please review

**Pull request merged**